hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/TestCloseContainerEventHandler.java

```diff
@@ -154,16 +152,15 @@ public void testCloseContainerWithDelayByLeaseManager()
         pipelineManager, containerManager, scmContext,
         mockLeaseManager, timeoutInMs);
     closeHandler.onMessage(container.containerID(), eventPublisher);
-    Mockito.verify(mockLeaseManager, atLeastOnce())
-        .acquire(any(), anyLong(), any());
+    verify(mockLeaseManager, atLeastOnce()).acquire(any(), anyLong(), any());
     assertThat(leaseList.size()).isGreaterThan(0);
+    assertTrue(leaseList.size() > 0);
```

Comment:
`assertTrue` is duplicate here:

```suggestion
    assertThat(leaseList.size()).isGreaterThan(0);
```

Reply:
Done.